modules/EmergencyMedicine/src/EmergencyMedicine/Controller/prescriptionLetterController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace EmergencyMedicine\Controller;
+
+
+
+use ClinikalAPI\Model\GetTemplatesServiceTable;
+use FhirAPI\FhirRestApiBuilder\Parts\ErrorCodes;
+use GenericTools\Model\FormEncounterTable;
+use GenericTools\Model\Lists;
+use GenericTools\Model\ListsOpenEmrTable;
+use GenericTools\Model\ListsTable;
+use Interop\Container\ContainerInterface;
+use ClinikalAPI\Controller\PdfBaseController;
+
+class prescriptionLetterController extends PdfBaseController
+{
+    const CATEGORY = ""; //Referral for Summary
+    const BODY_PATH = ['emergency-medicine/prescription-letter/prescription-letter'];
+
+
+
+    public $container = null;
+    public function getPregnancyState(){
```

Comment:
all the code here is from xray refferal letter and need to remve, right?

Reply:
as I already told you this is a promo demo code for task EM-100 do whatever you want with it .